cmd/asset-syncer/cmd/root_test.go

```diff
@@ -18,8 +18,9 @@ func TestParseFlagsCorrect(t *testing.T) {
 		conf server.Config
 	}{
 		{
-			"all arguments are captured (root command)",
```

Comment:
The root command can only take a sub command, so testing the invalidate instead.